gnovm/cmd/gno/test.go

```diff
@@ -638,6 +638,7 @@ func loadTestFuncs(pkgName string, t *testFuncs, tfiles *gno.FileSet) *testFuncs
 func parseMemPackageTests(memPkg *std.MemPackage) (tset, itset *gno.FileSet) {
 	tset = &gno.FileSet{}
 	itset = &gno.FileSet{}
+	errors := []error{}
```

Comment:
We typically use `multierr` for this. Can you switch to it or explain why your solution is more suitable in this case?

Reply:
I'll try to use multiErr I think it will fit perfectly

Reply:
fixed on 18fe895c2e5c05af7a5f3ff804ac9da746bd39df